README.md

```diff
@@ -65,24 +65,36 @@ If you used a different email address for your Amazon Developer account, follow
 Game Flow
 ---------
 
-Player perspective:
+### Player perspective
 ```
 Choose attack -> Hear opponent's attack -> Choose defense
-^                                           |
-|                                           v
-|-- If game continues <- Hear result of all combat -> If victory or defeat,
+^                                           │
+│                                           v
+└── If game continues <- Hear result of all combat -> If victory or defeat,
                                                       hear end of game info
 ```
 
-Engine perspective:
-```
-Give player two attack choices
-Player chooses attack action
-Choose opponent's attack (randomly for now)
-Speak opponent's attack to player and give player two defense choices
-Player chooses defense action
-Calculate if victory has occurred -> if victory or defeat, give player end of game info
-Calculate combat results
-Give player important combat results
-^ Back to top ^
-```
+### Engine perspective
+1. Give player two attack choices. Choices are from the player's deck and have offense === true
+2. Player chooses attack action
+3. Choose opponent's attack (randomly for now)
+4. Speak opponent's attack to player and give player two defense choices. Choices are from the player's deck and have `defense === true` and ship or planet === opponent's attack ship or planet
+5. Player chooses defense action
+6. Calculate if victory has occurred, if victory or defeat, skip to 10
+7. Calculate combat results. See next section.
+8. Give player important combat results
+9. ^ Back to 1 ^
+10. Give player end of game info
+
+### Combat mechanics
+* Attacks go to the planet or to the opponent's ship.
+* Defense goes to the planet or to your own ship.
+* Check each ship and the planets, three "combat zones", to see if victory has occurred
+   * If this is a ship and the defense has 0 cards at the ship, the ship takes one damage. 2 damage and the ship is destroyed and the offensive team wins with a ship victory.
+   * If this is the planet and one side has 0 cards, the other side is now "entrenched". If they were already entrenched, they win with a planet victory.
```

Comment:
Hrm.  You are right and I knew that when I wrote it but wrote it anyways.  I will fix it quickly and just push out a change.